docs/eggs/publishing-a-module.md

```diff
@@ -26,10 +54,10 @@ jobs:
       - uses: denolib/setup-deno@master
         with:
           deno-version: 1.1.1
-      - run: deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.1.8/mod.ts
+      - run: deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.2.0/mod.ts
```

Comment:
Side note: We should also update this on the `eggs-ci` repository.

Reply:
CC @maximousblk